Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -223,7 +223,7 @@ class PlaceDetailsCardViewController: UIViewController {
         if !OpenInHelper.open(url: url) {
             print("lol unable to open web address")
         } else {
-            Analytics.logEvent(event: AnalyticsEvent.EVENT_BANNER_LINK, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.CLICKED])
+            Analytics.logEvent(event: AnalyticsEvent.EVENT_BANNER_LINK, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.EVENT_CLICKED])
```

Comment:
It's strange that `EVENT_CLICKED` is an event in the previous use case but `EVENT_CLICKED` here is a param.